### PR TITLE
Missing bullet dependencies for building

### PR DESCRIPTION
**Pull request by j-rivero**
Should fix problems for https://github.com/ignition-tooling/release-tools/pull/420

### CODE REVIEW

**Comment by j-rivero** [comment]
Testing it here [Build Status](https://build.osrfoundation.org/job/ign-physics4-debbuilder/257/)

**Review by adlarkin — COMMENTED** [review]
Do we need to make the same change for `focal/debian/control`?

**Comment by j-rivero** [comment]
> Do we need to make the same change for `focal/debian/control`?

ouch, good catch. 

[Build Status](https://build.osrfoundation.org/job/ign-physics4-debbuilder/258/). Ouch the release-tools PR is broken.

Here is one using master:
[Build Status](https://build.osrfoundation.org/job/ign-physics4-debbuilder/259/)

File: focal/debian/control [hunk 1]
```diff
@@ -15,6 +15,7 @@ Build-Depends: cmake,
                libignition-utils1-dev,
                libignition-utils1-cli-dev,
                libdart6-dev (<< 6.10.0) | libdart-dev (>> 6.9.0),
+               libdart6-collision-bullet-dev (<< 6.10.0) | libdart-collision-bullet-dev (<< 6.10.0),
```
Comment (adlarkin) [inline focal/debian/control]:
```suggestion
               libdart6-collision-bullet-dev (<< 6.10.0) | libdart-collision-bullet-dev (>> 6.9.0),
```
Should this match what was done in #7?

**Pull request merged**